<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowWave</class>
 <widget class="QMainWindow" name="ShowWave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>875</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/images/wave.png</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="showWaveBtn">
        <property name="text">
         <string>显示波形</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pauseWaveBtn">
        <property name="text">
         <string>暂停波形</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearWaveBtn">
        <property name="text">
         <string>清除波形</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="drawGridBtn">
        <property name="text">
         <string>显示网格</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveWaveBtn">
        <property name="text">
         <string>保存波形</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadFileBtn">
        <property name="text">
         <string>加载文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBoxChSelect">
          <property name="title">
           <string>通道选择</string>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>61</width>
             <height>84</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="ch1CheckBox">
              <property name="text">
               <string>通道１</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ch2CheckBox">
              <property name="text">
               <string>通道２</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ch3CheckBox">
              <property name="text">
               <string>通道３</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ch4CheckBox">
              <property name="text">
               <string>通道４</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxFrameConfig">
          <property name="title">
           <string>帧首配置</string>
          </property>
          <widget class="QWidget" name="layoutWidget1">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>135</width>
             <height>172</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_1">
              <property name="text">
               <string>CH1帧首</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="frame_ch1">
              <property name="text">
               <string>X:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>CH2帧首</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="frame_ch2">
              <property name="text">
               <string>Y:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>CH3帧首</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="frame_ch3">
              <property name="text">
               <string>Z:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>CH4帧首</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="frame_ch4"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QwtPlot" name="wavePlot"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="actionShow"/>
    <addaction name="actionStop"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionShow">
   <property name="text">
    <string>&amp;Show</string>
   </property>
   <property name="toolTip">
    <string>Show Wave</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop show wave</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear old wave</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
